cpp/include/Ice/ProxyFunctions.h

```diff
@@ -201,59 +201,6 @@ namespace Ice
      */
     ICE_API bool
     proxyIdentityAndFacetEqual(const std::optional<ObjectPrx>& lhs, const std::optional<ObjectPrx>& rhs) noexcept;
-
-    /**
-     * A functor that compares the object identities of two proxies. Evaluates true if the identity in lhs
```

Comment:
Removed because we don't use these partially undocumented functors anywhere at all.